client/src/app/services/rsvp.service.ts

```diff
@@ -5,13 +5,17 @@ import { ApiService } from './api.service';
 
 @Injectable()
 export class RsvpService {
-  constructor(private api: ApiService) {}
+	constructor(private api: ApiService) {}
 
-  sendRSVP(data: RecursivePartial<submissionData>) {
-	return this.api.post(`rsvp`, data);
-  }
+	sendRSVP(data: RecursivePartial<submissionData>) {
+		return this.api.post(`rsvp`, data);
+	}
 
-  getRSVPFromCode(code: string) {
-	return this.api.get(`rsvp/${code}`);
-  }
+	getRSVP(code: string | null) {
```

Comment:
For future reference, you could just do

```suggestion
	getRSVP(code?: string) {
```

For optional parameters

Reply:
o7 weird differences between null & undefined, will leave for now